### PR TITLE
Adding new animation demo to show what can be done when using set_dig…

### PR DESCRIPTION
**Pull request by geekguy-wy**
This adds a new animation demo that shows a little bit of what can be done when using set_digit_raw()

### CODE REVIEW

**Comment by ladyada** [comment]
hiya please lint and then retag us to review

**Comment by geekguy-wy** [comment]
Done and ready for review.

File: examples/ht16k33_animation_demo.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        Param: bitmasks - a list of the bitmasks to write, in sequence, to the specified digits.
+        Param: delay - The delay, in seconds (or fractions of), between writing bitmasks to a digit.
+        Param: cycles - The number of cycles (loops) to write an animation.
+
+        Returns: No result
+    '''
+    cy = 0
+
+    while cy < cycles:
+    # cy in range(cycles):
```
Comment (makermelissa) [inline examples/ht16k33_animation_demo.py:48]: Could you either make this comment a little more clear or if it's commented out code, please remove. Thanks

File: examples/ht16k33_animation_demo.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        animate([0], [E, F, H, G2, 0], delay)
+        animate([1, 2], [G1, G2, 0], delay)
+        animate([3], [G1, J, A, 0], delay)
+        animate([2, 1], [A, 0], delay)
+        animate([0], [A, F, E, D, 0], delay)
+        animate([1, 2], [D, 0], delay)
+        animate([3], [D, C, B, J, G1, 0], delay)
+        animate([2, 1], [G2, G1, 0], delay)
+        animate([0], [H, 0], delay)
+        #animate([0], [M, H, 0x0], delay)
```
Comment (makermelissa) [inline examples/ht16k33_animation_demo.py:73]: Is there a reason this is commented out?

File: examples/ht16k33_animation_demo.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                display.set_digit_raw(dig, bits)
+                sleep(delay)
+
+        cy += 1
+
+def chase_animation(delay=DEFAULT_CHAR_DELAY_SEC, cycles=DEFAULT_CYCLES):
+    cy = 0
+
+    while cy < cycles:
+    #for cy in range(cycles):
```
Comment (makermelissa) [inline examples/ht16k33_animation_demo.py:60]: Let's remove any commented out code that is not intended to be used.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Looks like there are a few places with commented out code. Otherwise it looks good.

File: examples/ht16k33_animation_demo.py (hunk 1 above)
Comment (geekguy-wy) [reply examples/ht16k33_animation_demo.py:48]: I removed some commented out code and comments that were not needed.

File: examples/ht16k33_animation_demo.py (hunk 2 above)
Comment (geekguy-wy) [reply examples/ht16k33_animation_demo.py:73]: It was an alternate ending to the animation I thought I might use. Removed.

File: examples/ht16k33_animation_demo.py (hunk 3 above)
Comment (geekguy-wy) [reply examples/ht16k33_animation_demo.py:60]: Done. I have a new version that has some more complex animations.

**Comment by makermelissa** [comment]
If PR #60 supersedes this PR, then please close this one. They both have the animation demo. Thanks.

**Comment by geekguy-wy** [comment]
This PR is superseded by PR #60.

**Pull request closed without merging**